infra/modules/azure_app_service/app_service_slot.tf

```diff
@@ -22,7 +22,7 @@ resource "azurerm_linux_web_app_slot" "this" {
     ip_restriction_default_action     = "Deny"
 
     application_stack {
-      node_version = var.stack == "node" ? var.node_version : null
+      node_version = var.stack == "node" ? "${var.node_version}-lts" : null
```

Comment:
Why this change?

Reply:
After a plan i noticed that it gave an error because it wanted a string like `"20-tls"` like in `app_service.tf`, but instead it had a number.

Reply:
@lucacavallaro did you notice this issue when applied [this](https://github.com/pagopa/io-fims/blob/cba305f01699061ad0e69fb1bb2034bac372b353/infra/resources/_modules/web_apps/linux_function_app_op.tf#L12)?

Reply:
@mamu0 go ahead with `-lts` in both modules
Close this conv when you read it